shortcodes.php

```diff
@@ -380,56 +380,58 @@ function sc_org_chart($attrs) {
 		'orderby'     => 'menu_order',
 		'order'       => 'ASC',
 	)) : false;
-	
-	function display_people($category){
-		$people = get_posts(array(
-			'numberposts' => -1,
-			'post_type'   => 'profile',
-			'category'    => $category->term_id,
-			'orderby'     => 'menu_order',
-			'order'       => 'ASC',
-		));
 
-		?>
-		<div class="row">
-		<?
-		$count = 0;
-		foreach($people as $person) {
-			if($count > 0 && ($count % 4) == 0) {
-				echo '</div><div class="row">';
-			}
-			$department_text = get_post_meta($person->ID, 'profile_department_text', True);
-			$department_url  = get_post_meta($person->ID, 'profile_department_url', True);
-			$email           = get_post_meta($person->ID, 'profile_email', True);
+	if ( !function_exists( 'display_people' ) ) {
```

Comment:
If this is the same function as what's nested within the shortcode (https://github.com/UCF/EVP-AFIA-Bootstrap/blob/master/shortcodes.php#L384), I'd suggest removing it from the shortcode entirely; it would probably resolve the need to add a functions_exists() check here.

If things don't work after removing the function from the shortcode, what you've added here looks good.